addons/base_vat/base_vat.py

```diff
@@ -8,6 +8,12 @@
 _logger = logging.getLogger(__name__)
 
 try:
+    from suds.client import Client
+except ImportError:
+    _logger.warning("Automatic address filling for partner using VIES service would not be possible as `suds` Python library cannot be found. "
+                                          "Install it to enable this feature, for example with `easy_install suds`.")
+    Client = None
```

Comment:
Since V9, `suds` is no more an optional but required dependency (https://github.com/odoo/odoo/commit/6dfbca2ba06b4f2b3d908509b6b4ae1d44e0cc15) so this warning is useless...